tests/product/standalone/presto_installer.py

```diff
@@ -137,11 +118,7 @@ def _detect_presto_rpm():
             # are multiple RPMs, the last one is probably the latest
             rpm_name = sorted(rpm_names)[-1]
         else:
-            try:
-                rpm_name = StandalonePrestoInstaller._download_rpm()
-            except:
-                # retry once
-                rpm_name = StandalonePrestoInstaller._download_rpm()
+            raise OSError(1, 'Presto RPM not detected.')
```

Comment:
nit: you have a testcase where this gets called in the constructor. Does it make sense to pass it in here and testcase.fail() if the RPM doesn't exist?

It might not because it isn't a real test case, if it's messier that way, I'm fine with it as-is.